docs/source/tutorial.rst

```diff
@@ -42,14 +42,32 @@ To use a loader, (for example, ``urbansound8k``) you need to initialize it by ca
 You can indicate where the data would be stored and access by passing a path to ``data_home``, as explained below. Now ``us8k`` is a ``Dataset``
 object containing common methods, described in the following.
 
+Some datasets have more that one version available. Soundata can handle more than one version if included in the dataloader.
+
+.. code-block:: python
+
+    import soundata
+    dataset = soundata.initialize('urbansound8k', data_home='choose_where_data_live', version="1.0")
+
+By default, version is set to ``'default'``, which routes to the latest version available in the dataloader. 
```

Comment:
That's right @hagenw. I was trying to mention that we point to the _highest_ version. Therefore, in the example you posed, we default would still point to 2.0. I actually wonder if it is actually right to assume that we should we pointing to the highest version, probably not. It will most likely depend on each case. 

One thing that we could do, is letting the data loader creator to decide which version should be defaulted, and then update the `list_dataset_versions()` so that instead of returning the list of version numbers, displays a message like:

```
Dataset <dataset-name> has the following versions available: ["1.0", "1.1", "2.0"].
Default version for <dataset-name> is "2.0".
```
Does that sound good? Of course we would document these change as such.

Reply:
Nonetheless, if we want a criteria for that issue we could ask the contributors to default to the highest version unless there is a motivation to point to a different one (maybe the highest one is super new and most of the works are based on the old one, etc.)

Reply:
I think allowing to set a default version by the dataset maintainer makes sense, otherwise you cannot choose a default with a lower version number.